verifier/src/report_body.rs

```diff
@@ -878,4 +889,18 @@ mod test {
             1
         );
     }
+
+    #[test]
+    fn attributes_verifier_error_display() {
```

Comment:
:wrench: This doesn't look to be "error" related. Maybe the drive for this change was to display the verifier in error circumstances, but this looks to be testing the display, not necessarily errors
```suggestion
    fn attributes_verifier_display() {
```